src/pydrex/diagnostics.py

```diff
@@ -279,11 +284,16 @@ def misorientation_indices(
     The `orientation_stack` is a NxMx3x3 array of orientations where N is the number of
     texture snapshots and M is the number of grains.
 
-    Uses the multiprocessing library to calculate texture indices for multiple snapshots
-    simultaneously. If `ncpus` is `None` the number of CPU cores to use is chosen
-    automatically based on the maximum number available to the Python interpreter,
-    otherwise the specified number of cores is requested. Alternatively, an existing
-    instance of `multiprocessing.Pool` can be provided.
+    Uses either Ray or the Python multiprocessing library to calculate texture indices
+    for multiple snapshots simultaneously. The arguments `ncpus` and `pool` are only
+    relevant the latter option: if `ncpus` is `None` the number of CPU cores to use is
+    chosen automatically based on the maximum number available to the Python
+    interpreter, otherwise the specified number of cores is requested. Alternatively, an
+    existing instance of `multiprocessing.Pool` can be provided.
+
+    If Ray is installed, it will be automatically preferred. In this case, the number of
+    processors (actually Ray “workers”) should be set upon initialisation of the Ray
```

Comment:
I do not think that `workers` is specific to Ray. Are you trying to oppose multiprocessing `processes` to Ray `Actors`?

Reply:
changed wording